src/containers/PageRoot.tsx

```diff
@@ -173,10 +211,18 @@ class PageRoot extends ApiComponent<
               </Col>
             </Row>
           </div>
-        </Header>
+        </Header>}
         <Layout>
-          <Sider collapsible width={200} style={{ background: "#fff" }}>
+          <Sider 
+            breakpoint="lg" 
+            collapsible 
+            width={200} 
+            collapsedWidth={self.props.isMobile ? 0 : 50} 
```

Comment:
Default `collapsedWidth` should be 80, not 50, otherwise, in desktop mode, when you collapse the sidebar you see a broken layout:

![screenshot from 2019-02-09 10-15-38](https://user-images.githubusercontent.com/10582082/52524620-b0b3ec80-2c53-11e9-910b-8a9f271921d1.png)